lib/test/ship_test.rb

```diff
@@ -0,0 +1,63 @@
+require 'minitest/autorun'
```

Comment:
Working on moving files now!

Reply:
Also - great catch. Thank you :)

Reply:
Note to self - [Article](https://openclassrooms.com/en/courses/4614926-learn-the-command-line-in-terminal/4634381-move-and-copy-items-in-your-system) on moving files using terminal. Don't forget to use git rm <filename> instead of git add as you delete files or directories.